webpack/assets/javascripts/react_app/components/Bookmarks/components/__tests__/SearchModal.test.js

```diff
@@ -1,12 +1,13 @@
 import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';
 
-import SearchModal from '../SearchModal';
+import SearchModal from '../SearchModal/SearchModal';
 
 const props = {
   controller: 'hosts',
   show: true,
```

Comment:
Show prop is not used in the new SearchModal so it shouldn't be passed or tested

Reply:
@MariaAga Thanks, updating